draft-ietf-quic-transport.md

```diff
@@ -3835,107 +3849,148 @@ fields.  The PMTU can depend on path characteristics, and can therefore change
 
 The requirements for generating ICMP ({{?RFC1812}}, {{?RFC4443}}) state that the
 quoted packet should contain as much of the original packet as possible without
 exceeding the minimum MTU for the IP version.  The size of the quoted packet can
 actually be smaller, or the information unintelligible, as described in Section
 1.1 of {{!DPLPMTUD}}.
 
-QUIC endpoints SHOULD validate ICMP messages to protect from off-path injection
-as specified in {{!RFC8201}} and Section 5.2 of {{!RFC8085}}. This validation
-SHOULD use the quoted packet supplied in the payload of an ICMP message to
-associate the message with a corresponding transport connection {{!DPLPMTUD}}.
-
-ICMP message validation MUST include matching IP addresses and UDP ports
-{{!RFC8085}} and, when possible, connection IDs to an active QUIC session.
-
-The endpoint SHOULD ignore all ICMP messages that fail validation.
+QUIC endpoints using PMTUD SHOULD validate ICMP messages to protect from
+off-path injection as specified in {{!RFC8201}} and Section 5.2 of {{!RFC8085}}.
+This validation SHOULD use the quoted packet supplied in the payload of an ICMP
+message to associate the message with a corresponding transport connection (see
+Section 4.6.1 of {{!DPLPMTUD}}).  ICMP message validation MUST include matching
+IP addresses and UDP ports {{!RFC8085}} and, when possible, connection IDs to an
+active QUIC session.  The endpoint SHOULD ignore all ICMP messages that fail
+validation.
 
 An endpoint MUST NOT increase PMTU based on ICMP messages; see Section 3, clause
 6 of {{!DPLPMTUD}}.  Any reduction in the QUIC maximum packet size in response
 to ICMP messages MAY be provisional until QUIC's loss detection algorithm
 determines that the quoted packet has actually been lost.
 
 
-## Datagram Packetization Layer PMTU Discovery
+## Datagram Packetization Layer PMTU Discovery {#dplpmtud}
+
+Datagram Packetization Layer PMTU Discovery (DPLPMTUD;
+{{!DPLPMTUD=I-D.ietf-tsvwg-datagram-plpmtud}}) relies on tracking loss or
+acknowledgment of QUIC packets that are carried in PMTU probes.  PMTU probes for
+DPLPMTUD that use the PADDING frame implement "Probing using padding data", as
+defined in Section 4.1 of {{!DPLPMTUD}}.
+
+Endpoints SHOULD set the initial value of BASE_PMTU (see Section 5.1 of
+{{!DPLPMTUD}}) to be consistent with the minimum QUIC packet size. The
+MIN_PLPMTU is the same as the BASE_PMTU.
+
+QUIC endpoints implementing DPLPMTUD maintain a maximum packet size (DPLPMTUD
+MPS) for each combination of local and remote IP addresses.
 
-Section 6.3 of {{!DPLPMTUD}} provides considerations for implementing Datagram
-Packetization Layer PMTUD (DPLPMTUD) with QUIC.
+If a QUIC endpoint determines that the PLPMTU between any pair of local and
+remote IP addresses has fallen below the size needed to support the minimum QUIC
+packet size (BASE_PLPMTU), it MUST immediately cease sending QUIC packets on the
+affected path. An endpoint MAY terminate the connection if an alternative path
```

Comment:
This seems a bit repetitive of 3862-3866.  It's also still confusing or potentially contradictory to me.